lib/services/local/access_list.go

```diff
@@ -138,6 +138,11 @@ func (a *AccessListService) GetAccessLists(ctx context.Context) ([]*accesslist.A
 	return accessLists, trace.Wrap(err)
 }
 
+// GetInheritedGrants returns grants inherited from parent access lists. This is not implemented in the local service.
+func (a *AccessListService) GetInheritedGrants(ctx context.Context, accessListID string) (*accesslist.Grants, error) {
+	return nil, trace.NotImplemented("GetInheritedGrants should not be called")
```

Comment:
I think to satisfy the `AccessListService` interface, and since it's not used in the backend service? It's implemented in `api/client/accesslist/accesslist.go`:
https://github.com/gravitational/teleport/blob/3565893b08be6b0ec433b53cc50316ee86b971ba/api/client/accesslist/accesslist.go#L118-L130